build/js/live-editor.output_pjs.js

```diff
@@ -1602,10 +1614,38 @@ window.PJSOutput = Backbone.View.extend({
             // but PImage may be mutable, so that might not work.
             this.imageCache[file] = img;
         }.bind(this));
-    },
+        
+        _.each(sounds, function(file) {
+            var fileMatch = /["']([A-Za-z0-9_\/-]*?)["']/.exec(file);
 
+            if (!fileMatch) {
+                return loaded();
+            }
 
-    soundCache: {},
+            file = fileMatch[1];
+            
+            if (this.soundCache[file]) {
```

Comment:
I thought about it a little more and I'll do one loop to get all references and then a second loop to do initial all of the ```GET``` request.  That way we can display a message like 3 of 4 resources loaded.